packages/integration-tests/playwright.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig, devices } from "@playwright/test";
+import { config } from "tests/ui/config";
+
+export default defineConfig({
+  testDir: "tests/ui",
+  fullyParallel: true,
+  projects: [
+    {
+      name: "chromium",
```

Comment:
Why the project is called "chromium"? Should dit be just "integration-tests" or something?

Reply:
updated